recipes/go-task/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set goname = "github.com/go-task/task" %}
+{% set name = "go-task" %}
+{% set version = "3.24.0" %}
+
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  - folder: {{ pkg_src }}
+    url: https://{{ goname }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: e1ab3c92299ad5741b8b0e9dc18593db80efaca449b8ccc318a6ff1848b44338
+
+build:
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - go install -v -ldflags="-w -s -X main.version=conda-forge" ./cmd/task
+    - go-licenses save "." --save_path "library_licenses"
+
+requirements:
+  build:
+    - {{ compiler('go') }}
```

Comment:
Please use either go-nocgo or go-cgo here.

Reply:
I added `go-nocgo` but kept the `{{ compiler('go') }}` as I was not working otherwise.